### DISCUSSION TITLE
data-importer vs csv-importer

### DISCUSSION THREAD

**Discussion by MacPaille** [firefly-iii/firefly-iii#5448] [General]
Hey JC5,

Just to be sure to understand after having read all the doc available (I may have missed it somewhere...) what is the difference between the data-importer and csv-importer?
It seems FIDI is just the CSV-importer with the feature to be able to directly access the bank accounts via Spectre or Nordigen. Am I right?

Can I still use FIDI the same way I'm currently using CSV-importer?

Thanks!

**Comment by JC5** [comment]
Yes, and yes.

**Reply by JC5** [reply]
Basically I merged the three importers into one to save a lot of time debugging and double code.

**Reply by MacPaille** [reply]
Super good idea and I already love the possibility to directly fetch in json from the bank. I'm looking into it now :)
The more I use your tool the more I love it :)
